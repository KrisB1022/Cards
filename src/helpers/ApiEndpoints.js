export const baseUrl = "https://api.magicthegathering.io";

export const cardsEndpoint = `${baseUrl}/v1/cards`;
export const typesEndpoint = `${baseUrl}/v1/types`;
