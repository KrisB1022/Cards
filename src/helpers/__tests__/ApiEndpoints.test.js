import * as endpoints from "../ApiEndpoints";

it("matches snapshot", () => {
	expect(endpoints).toMatchSnapshot();
});
