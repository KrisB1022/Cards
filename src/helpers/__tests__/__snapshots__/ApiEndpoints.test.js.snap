// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "baseUrl": "https://api.magicthegathering.io",
  "cardsEndpoint": "https://api.magicthegathering.io/v1/cards",
  "typesEndpoint": "https://api.magicthegathering.io/v1/types",
}
`;
